packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.styles.ts

```diff
@@ -53,7 +53,10 @@ export const styles = (props: IDatePickerStyleProps): IDatePickerStyles => {
           pointerEvents: 'initial',
           cursor: 'pointer'
         }
-      ]
+      ],
+      disabled && {
+        color: semanticColors.disabledText
```

Comment:
Would really appreciate if you can take a look into adding a screener test for a disabled DatePicker to avoid future regressions. It should be really straight forward to add them following the other stories in here. Let us know if you have issues with it and we maybe can add it as a separate PR.
https://github.com/OfficeDev/office-ui-fabric-react/blob/f8f23e10e9fa0c9b92111e108bb2957fab82ada1/apps/vr-tests/src/stories/DatePicker.stories.tsx#L38-L62

Reply:
I'll pick this up tomorrow when I'm back in the office. It looks easy enough but if I can't figure it out I'll let you know.

Reply:
I've attempted to add test/stories for the disabled state, and they seem to work, I guess someone has to approve the screenshots?